.github/workflows/build.yml

```diff
@@ -236,7 +236,7 @@ jobs:
 
     steps:
       - name: Notify
-        uses: tsickert/discord-webhook@v5.3.0
+        uses: tsickert/discord-webhook@v6.0.0
```

Comment:
Please do these separate version bumps in a different PR.

Reply:
will do it no problem